src/main/python/trec-covid/print_run.py

```diff
@@ -0,0 +1,42 @@
+"""Script to print TREC runs into a human readable format"""
+import argparse
+import csv
+import utils
+from tqdm import tqdm
+
+
+def load_metadata(path):
+    metadata = {}
+    with open(path) as f:
+        for row in tqdm(csv.DictReader(f)):
+            metadata[row['cord_uid']] = row['title'], row['abstract']
```

Comment:
In the future, we might need the value field to be an object with more info:

metadata[row['cord_uid']] = {
    "title": row['title'],
    ...
}

Reply:
Yeah, makes sense!